source/index.pug

```diff
@@ -1,20 +1,35 @@
 extends layout/layout
 block head
   +head("Home", true)
-block header
-  +header
 block content
+  +header
   main
     section.slider
       .container
         .row.no-gutters
           .col-sm-8.d-none.d-xl-block
             .slider__left
```

Comment:
1 div thừa?

Reply:
cái main thừa hay cái nào ạ? cái thẻ **main** kia để bao toàn bộ nội dung của trang index

Reply:
Chị đang comment dòng 12. Em phải để ý xem comment nằm ở dưới dòng nào thì nó là comment của dòng đó

Reply:
dòng 12 này không phải div thừa ạ. cái này chứa background của slider. trong PSD không có PSD chi tiết của phần slider mà có mỗi 1 layer đại diện cho slider thôi ạ

Reply:
Nếu là slider thì sao lại dùng background? Phải thêm image chứ?